srtcore/core.cpp

```diff
@@ -958,6 +960,34 @@ void CUDT::setOpt(SRT_SOCKOPT optName, const void* optval, int optlen)
         }
         break;
 
+      // This option is meaningless for the socket itself.
+      // It's set here just for the sake of setting it on a listener
+      // socket so that it is then applied on the group when a
+      // group connection is configuired.
+    case SRTO_GROUPSTABTIMEO:
```

Comment:
Align comments similar to the "case ..." block.

Reply:
case labels are aligned equally to the braces of `switch`, so it's hard to say how surrounding comments should be aligned. I'd state they should be aligned as they are, but in case when they are under the label. Over the label this is still according to the rules, but I agree it looks stupid. Therefore I simply move it closer to the code so that this alignment looks more sense.